### ISSUE TITLE
Improve accessibility of account-ui

### ISSUE THREAD

**Issue by Evodam** [keycloak/keycloak#36479]
### Description

As far as I can see account-ui and admin-ui lack several accessibility features like:

-  use <p> for paragraphs instead of div
-  assign error message to their respective form field with aria attributes
-  group form fields and labels with <fieldset>

### Discussion

https://github.com/keycloak/keycloak/discussions/36474

**Comment by Evodam** [comment]
@ssilvert 
@agagancarczyk 
you removed triage label. Does this mean this request is accepted? Could I contribute and prepare a pull request with the needed changes?
